appveyor.yml

```diff
@@ -51,7 +51,7 @@ before_build:
 - cmd: set CPATH=%LIBELF_INCLUDE%;%CPATH%
 - cmd: set CPLUS_INCLUDE_PATH=%LIBELF_INCLUDE%;%BOOST_ROOT%;%CPLUS_INCLUDE_PATH%
 # set variables for MSVC
-- cmd: if "%build%"=="msvc"  call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" %PLATFORM%
```

Comment:
It is required to add this line before before:
```yaml
-cmd: if "%build%"=="msvc"  "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat"
```

Reference: http://help.appveyor.com/discussions/kb/48-vs2017